app/BUILD.bazel

```diff
@@ -807,6 +810,8 @@ crashlytics_android_library(
 android_library(
     name = "crashlytics_deps",
     exports = [
+        "//third_party:androidx_core_core",
+        "//third_party:androidx_core_core-ktx",
```

Comment:
This doesn't seem correct--why do we need to export these here?

Reply:
Removed this library from here.

Reply:
There isn't enough information from that screenshot to see which dependency is failing to build. Building it locally showed me that the only dep changes needed for this PR:
- Adding androidx_core_core to view_models
- Adding androidx_core_core to TEST_DEPS (this might not actually be needed, but prefer adding it)

None of the other changes are needed (including the new core-ktx as they aren't used).

I'm not actually 100% sure why view_models and databinding libraries need this--that's not expected. There might be some weird dependency swapping happening here by maven_install, but I can't be sure. Either way, the changes above seem reasonable.

Reply:
This isn't fully resolved yet--PTAL at my latest comment.